src/storage/tile.cpp

```diff
@@ -243,6 +239,8 @@ const std::string Tile::GetInfo() const {
   return os.str();
 }
 
+void Tile::CompressTile(Tile *tile) { (void)tile; }
```

Comment:
You can use UNUSED_ATTRIBUTE to disable the warning. `void Tile::CompressTile(UNUSED_ATTRIBUTE Tile *tile)`